src/widgets/_progressbars.scss

```diff
@@ -32,14 +32,43 @@ progressbar {
     }
 
     &.horizontal {
-        progress,
         trough {
             @extend %outset-background;
         }
 
         progress {
             margin-left: -1px;
             margin-right: -1px;
+
+            &:not(.pulse) {
+                background-size: cover, 48px 100%;
+                background-repeat: no-repeat;
+                background-image:
+                    linear-gradient(
+                        to bottom,
+                        #{'alpha(@highlight_color, 0.2)'},
+                        rgba(white, 0)
+                    ),
+                    linear-gradient(
+                        to right,
+                        rgba(white, 0),
+                        rgba(white, 0.25) 60%,
+                        rgba(white, 0)
+                    );
+
+                &:dir(ltr) {
+                    animation: progress 2s ease infinite;
+                }
+
+                &:dir(rtl) {
+                    animation: progress-rtl 2s ease infinite;
```

Comment:
I'm not sure if it's supported, but you might save yourself a bit of code by using `animation-direction: reverse`

https://developer.mozilla.org/en-US/docs/Web/CSS/animation-direction

Reply:
You so smart